src/prasad/FindNumber.java

```diff
@@ -0,0 +1,36 @@
+package prasad;
+
+import java.util.Scanner;
+
+/*Program 4 : Accepts array of numbers and finds +ve,-ve and 0 from the array*/
+public class FindNumber {
+	int posNo= 0;
+	int negNo= 0;
+	int zeros= 0;
+	void search(int[]nos) {
```

Comment:
no require to create instance level variable as you are directly printing them in to method.